apps/backend/src/data/migrations/1654703454488-create-category-modifiers-table.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey
+} from 'typeorm';
+
+export class CreateCategoryModifiersTable1654703454488
+  implements MigrationInterface
+{
+  async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'modifier-category',
+        columns: [
+          {
```

Comment:
`isNullable: true` means that this field can be null, which is not the case for this field. The only `isNullable: true` fields are  min and max selections in this table